.github/workflows/lint.yml

```diff
@@ -122,6 +122,7 @@ jobs:
                      --known-failure app/util/util.cpp \
                      --known-failure app/util/util.h \
                      --known-failure app/WriteHandler.h \
+                     --known-failure controller/webrtc/Callbacks.h \
```

Comment:
This list is only for file that are already in the SDK. Do not add new files which are not part of any BUILD.gn